components/kyma-environment-broker/internal/fixture.go

```diff
@@ -0,0 +1,257 @@
+package internal
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/kyma-project/control-plane/components/provisioner/pkg/gqlschema"
+
+	"github.com/pivotal-cf/brokerapi/v7/domain"
+
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/common/orchestration"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/ptr"
+	"k8s.io/utils/pointer"
+)
+
+const (
+	serviceId              = "47c9dcbf-ff30-448e-ab36-d3bad66ba281"
+	serviceName            = "kymaruntime"
+	planId                 = "4deee563-e5ec-4731-b9b1-53b42d855f0c"
+	planName               = "azure"
+	globalAccountId        = "e8f7ec0a-0cd6-41f0-905d-5d1efa9fb6c4"
+	provisionerOperationId = "e04de524-53b3-4890-b05a-296be393e4ba"
+	kymaVersion            = "1.19.0"
+)
+
+type SimpleInputCreator struct {
+	Overrides         map[string][]*gqlschema.ConfigEntryInput
+	Labels            map[string]string
+	EnabledComponents []string
+	ShootName         *string
+}
+
+func FixServiceManagerEntryDTO() *ServiceManagerEntryDTO {
+	return &ServiceManagerEntryDTO{
+		Credentials: ServiceManagerCredentials{
+			BasicAuth: ServiceManagerBasicAuth{
+				Username: "username",
+				Password: "password",
+			},
+		},
+		URL: "https://service-manager.local",
+	}
+}
+
+func FixERSContext(id string) ERSContext {
+	var (
+		tenantID     = fmt.Sprintf("Tenant-%s", id)
+		subAccountId = fmt.Sprintf("SA-%s", id)
+		userID       = fmt.Sprintf("User-%s", id)
+	)
+
+	return ERSContext{
+		TenantID:        tenantID,
+		SubAccountID:    subAccountId,
+		GlobalAccountID: globalAccountId,
+		ServiceManager:  FixServiceManagerEntryDTO(),
+		Active:          pointer.BoolPtr(true),
+		UserID:          userID,
+	}
+}
+
+func FixProvisioningParametersDTO() ProvisioningParametersDTO {
+	trialCloudProvider := TrialCloudProvider("provider")
+	return ProvisioningParametersDTO{
+		Name:                        "cluster-name",
+		TargetSecret:                pointer.StringPtr("TargetSecret"),
+		VolumeSizeGb:                ptr.Integer(50),
+		MachineType:                 pointer.StringPtr("MachineType"),
+		Region:                      pointer.StringPtr("Region"),
+		Purpose:                     pointer.StringPtr("Purpose"),
```

Comment:
why do we use a new pointer package? you can just use String function from our ptr package

Reply:
Good point, forgot to change that.